src/nvim/event/stream.c

```diff
@@ -97,6 +97,10 @@ void stream_close(Stream *stream, stream_close_cb on_stream_close, void *data)
   stream->close_cb = on_stream_close;
   stream->close_cb_data = data;
 
+#ifdef WIN32
+  uv_tty_set_mode(&stream->uv.tty, UV_TTY_MODE_NORMAL);
```

Comment:
`stream->uv` may not be a TTY stream, so we should check it somehow.

```suggestion
  if (UV_TTY == uv_guess_handle(stream->fd)) {
    uv_tty_set_mode(&stream->uv.tty, UV_TTY_MODE_NORMAL);
```

Reply:
It is now checked